papers/bradley_dice/55_bradley_dice.rst

```diff
@@ -0,0 +1,553 @@
+The growth in involvement with signac is the result of our focus on developing features based on user needs, as well as our efforts to transition signac users to signac contributors, through many initiatives in the past few years.
+Through encouraging users to become contributors, we ensure that signac addresses real users' needs.
+We have expanded signac's contributor involvement to outside of the University of Michigan through expanded use in diverse research groups (and through maintainers graduating and staying involved?), but more notably through the Google Summer of Code (GSoC) program.
+Our experience from the GSoC led to a new committer (explained later in this section) and much work on some of the developments presented above, namely synced collections and aggregation.
+To encourage code contributions from existing users, we maintain active support and discussion through Slack.
+In addition, we have started hosting weekly "office hours" for in-person (virtual) introduction and contributions to the code base.
+By pairing new contributors with experienced signac developers, we significantly reduce the knowledge barrier to joining a new project.
+Office hours creating space for users to make contributions has also led to more features and documentation born directly out of user need.
+Contributing to documentation has been a productive starting point for new users-turned-contributors, both for the users and the project, since it improves the users' familiarity with the API as well as addresses weak spots in the documentation more obvious to newer users.
+
+We will share our project's experience in progressively revising project governance to catalyze sustained contributions of many kinds, adding more points of entry for learning about the project (Slack support, office hours), and participating in Google Summer of Code in 2020 as a NumFOCUS Affiliated Project.
+
+In our growth with increasing contributors and users, we recognized a need to change our governance structure to make contributing easier and provide a clear organizational structure to the community.
+We based our new model on the Meritocratic Governance Model and our manager roles on Numba Czars.
+We decided on a four category system with maintainers, committers, contributors, and users.
+Code review and PR merge responsibilities are granted to maintainers and committers, who are (self-)nominated and accepted by a vote of the project maintainers.
+Contributors consist of all members of the community who have contributed in some way to the framework, which includes adding or refactoring code as well as filing issues and improving documentation.
+Finally, users refer to all those who use signac in any capacity.
```

Comment:
The definitions of contributors and users are somewhat obvious and I'm not sure if we need to state them for readers here.

Reply:
I think we should mention them in some capacity since we define the other two even if it is to say `Contributors and users represent those who contribute to and use the framework in any capacity.`

Reply:
This is good enough for me. Resolving.